src/ser.rs

```diff
@@ -19,7 +34,7 @@ pub struct Serializer<W, F = CompactFormatter> {
 
 impl<W> Serializer<W>
 where
-    W: io::Write,
+    W: write::Write,
```

Comment:
This is a breaking change. Please don't change the public API of Serializer in std mode.